rules/node.libsonnet

```diff
@@ -33,7 +33,7 @@
               count by (%(clusterLabel)s, node) (sum by (node, cpu) (
                 node_cpu_seconds_total{%(nodeExporterSelector)s}
               * on (namespace, %(podLabel)s) group_left(node)
-                node_namespace_pod:kube_pod_info:
+                topk by(namespace,pod) (1, node_namespace_pod:kube_pod_info:)
```

Comment:
Sorry that I didn’t think about this before but why don’t we fix this in the original recording rule here? I would expect this recording rule to always only have one entry per namespace pod combination.

Reply:
Ack, it makes sense.